FWCore/Services/web/transitions.css

```diff
@@ -0,0 +1,89 @@
+
+:root {
+--colr-black: #3C3530;
+--colr-green: #AACD6E;
+--colr-gray: #C5C6B6;
+}
+html {
+  box-sizing: border-box;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, "PingFang TC", "Heiti TC", 微軟正黑體, sans-serif;
+  font-size: 16px;
+  -webkit-font-smoothing: antialiased;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+	height: 30px;
+	float: bottom;
+	bottom:0;
+}
+.core {
+	height: 95vh;
+}
+.container {
+  display: flex;
+  width: 100%;
+  height: 100%;
+}
+
+.name_div {
+  width: 10%;
+  background-color: var(--colr-green);
+}
+#name_view {
+	flex:1;
+    width: 100%;
```

Comment:
Indentation seems to be inconsistent in some places of this file (some lines use tabs and others use spaces)

Reply:
fixed indention